src/test/java/splitStringClassTest.java

```diff
@@ -0,0 +1,71 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class splitStringClassTest {
```

Comment:
클래스 이름은 관례를 따라서 대문자로 시작하는 것이 좋겠습니다.
```suggestion
class SplitStringClassTest {
```